extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderController.class.php

```diff
@@ -69,6 +69,17 @@ public function getRedirectUrls() {
 		}
 	}
 
+	public function checkIfTemplateExists() {
```

Comment:
What would be a good alternative to this name, emphasizing that it is a 'get'?
Maybe getTemplateExistenceStatus or getTemplateStatus?